test/acceptance/features/bindAppToService.feature

```diff
@@ -416,7 +416,7 @@ Feature: Bind an application to a service
         And The application env var "HOST_ADDR" has value "127.0.0.1:8080"
 
     @olm
-    Scenario: Creating binding secret from the definitions managed in OLM operator descriptors
+    Scenario: Bind service to application using binding definition available in CSV x-descriptors 
```

Comment:
this scenario and the one in line 335 talk about the same thing? what is the difference?

Reply:
The use of x-descriptors is different.  Earlier one use:
`service.binding:ports:elementType=sliceOfMaps,sourceKey=protocol,sourceValue=port`

Reply:
but both of them define bindings via x-descriptors and they descriptions do not tell any other details:

* " Bind service to application using binding definition available in CSV x-descriptors "
* "Bind service to application using binding definition available in x-descriptors"

The descriptions are almost identical. Would it make sense to either remove one of the scenarios, or collapse them into single one?

Reply:
Let's remove the current one - it looks like a simple case.

Reply:
removed.